public/src/Helpers.ts

```diff
@@ -372,3 +312,30 @@ export const sortEnrollments = (enrollments: Enrollment[], sortBy: EnrollmentSor
         return enrollmentCompare(a, b, sortBy, descending)
     })
 }
+
+export class SubmissionsForCourse {
```

Comment:
Should this class be colocated with the `SubmissionOwner` type? Three alternatives:

- Separate new file `SubmissionsForCourse.ts`
- In this file (Helpers.ts)
- In state.tsx (where `SubmissionOwner` is defined)